cmd/snap/main.go

```diff
@@ -361,6 +361,7 @@ func Parser(cli *client.Client) *flags.Parser {
 	registerCommands(cli, parser, routineCommand, routineCommands, func(ci *cmdInfo) {
 		checkUnique(ci, "routine ")
 	})
+
```

Comment:
spurious?

Reply:
Yes, leftover from the removal. Removed it now, thanks